src/deployment/config.json

```diff
@@ -1,6 +1,12 @@
 {
+    "tenant_id": "72f988bf-86f1-41af-91ab-2d7cd011db47",
```

Comment:
I thought we were getting out of the business of hard coding these values.

Reply:
As a first step, I was just moving it out of the deploy script. This acts as an example tenant_id.

Reply:
I'd like to leave this in the first pass.